ui/summary.go

```diff
@@ -309,11 +212,11 @@ func (s *Summary) summarizeMetrics(w io.Writer, indent string, t time.Duration,
 
 				resolver := s.trendValueResolvers[tc]
 
-				switch v := resolver(sink).(type) {
-				case float64:
+				v := resolver(sink)
+				if tc != "count" { // sigh
```

Comment:
nitpick ... but why are you checking for the reverse here instead of for the equal one? ;)

Reply:
:man_shrugging: probably just because that was the case order in the old `switch`, but yeah, doesn't makes sense, so I'll  change it

Reply:
in the JS variant of this though, no need to further mess with the Go code